### PR TITLE
Added Rvision functionality

### PR DESCRIPTION
**Pull request by MurrayLF**

### CODE REVIEW

File: R/image_registration.R [hunk 1]
```diff
@@ -523,9 +639,14 @@ registerImagesFIJI = function(source,
...
-  source = path.expand(source)
-  output = path.expand(output)
+  if(file.exists(output)) {
+    unlink(output,recursive = TRUE)
+    dir.create(output)
+  }
+  
+  if(!file.exists(output)) {
+    dir.create(output)
+  }
```
Comment (MurrayLF) [inline R/image_registration.R]: Just want to make sure the this is the current method

File: R/image_registration.R [hunk 2]
```diff
@@ -442,10 +558,10 @@ fiji = function(fijiPath = NULL) {
       stop("fiji is not at: ", fijiPath)
   } else {
     # do we have an option set?
-    fijiPath=getOption('giotto.fiji')
+    fijiPath=getOption('jimpipeline.fiji')
```
Comment (MurrayLF) [inline R/image_registration.R]: Just want to make sure this is the correct name

File: R/image_registration.R (hunk 1 above)
Comment (jiajic) [reply R/image_registration.R]: Yeah that's a good addition probably.
The path.expand is still needed though. If a path is given like "~/" for example, that would be unreadable as part of a system command.

File: R/image_registration.R (hunk 2 above)
Comment (jiajic) [reply R/image_registration.R]: It should be giotto.fiji
That's something that Ruben changed.

**Review by jiajic — APPROVED** [review]
This is great thank you!
I'm going to merge these and then make any changes that got rolled back.

**Pull request merged**